src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1252,10 +1252,19 @@ elementFormDefault="qualified">
             <xs:documentation><!-- _locID_text="DebugSymbols" _locComment="" -->Whether to emit symbols (boolean)</xs:documentation>
         </xs:annotation>
     </xs:element>
-    <xs:element name="DebugType" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+    <xs:element name="DebugType" substitutionGroup="msb:Property">
         <xs:annotation>
-            <xs:documentation><!-- _locID_text="DebugType" _locComment="" -->none, pdbonly, or full</xs:documentation>
+            <xs:documentation><!-- _locID_text="DebugType" _locComment="" -->none, pdbonly, embedded, portable or full. From C# 6 onwards, pdbonly is the same as full.</xs:documentation>
```

Comment:
nit:
```suggestion
            <xs:documentation><!-- _locID_text="DebugType" _locComment="" -->none, pdbonly, embedded, portable, or full. From C# 6 onwards, pdbonly is the same as full.</xs:documentation>
```

Reply:
I was curious and this is indeed [standard Microsoft Docs style](https://docs.microsoft.com/en-us/style-guide/punctuation/commas#use-a-comma).

Reply:
I read the style docs!

(Once)

(About a year ago)

😋